docs/developer-guide/code-contributions.md

```diff
@@ -13,11 +13,6 @@ Before submitting code for a new feature (and also, to some extent, for more com
 [raise an Enhancement Proposal or Bug Issue](https://github.com/argoproj/argo-cd/issues/new/choose)
 first.
 
-Each enhancement proposal needs to go through our
-[triage process](#triage-process)
-before we accept code contributions. To facilitate triage and to provide transparency, we use
-[this GitHub project](https://github.com/orgs/argoproj/projects/18) to keep track of this process' outcome.
-
```

Comment:
The lines below also should be removed or updated.
I think the current process is to either implement something or discuss it on CNCF Slack or contributors meeting.